dfdaemon/config/config.go

```diff
@@ -247,29 +247,29 @@ func (u *URL) unmarshal(unmarshal func(interface{}) error) error {
 	return nil
 }
 
-// MarshalJSON implements json.Marshaller to print the url
+// MarshalJSON implements json.Marshaller to print the url.
 func (u *URL) MarshalJSON() ([]byte, error) {
 	return json.Marshal(u.String())
 }
 
-// MarshalYAML implements yaml.Marshaller to print the url
+// MarshalYAML implements yaml.Marshaller to print the url.
 func (u *URL) MarshalYAML() (interface{}, error) {
 	return u.String(), nil
 }
 
 // CertPool is a wrapper around x509.CertPool, which can be unmarshalled and
-// constructed from a list of filenames
+// constructed from a list of filenames.
```

Comment:
Uppercase `Consgructed`

Reply:
This line and the previous line are one sentence. 
Don't need to update.

Reply:
Sorry.